sopel/irc/backends.py

```diff
@@ -72,38 +96,39 @@ class AsynchatBackend(AbstractIRCBackend, asynchat.async_chat):
     :param bot: a Sopel instance
     :type bot: :class:`sopel.bot.Sopel`
     :param int server_timeout: connection timeout in seconds
-    :param int ping_timeout: ping timeout in seconds
+    :param int ping_interval: ping interval in seconds
     """
```

Comment:
Just in the spirit of improving our documentation's comprehensiveness whenever possible:

```suggestion

    The ``server_timeout`` option defaults to ``120`` seconds if not provided.

    The ``ping_interval`` defaults to ``server_timeout * 0.45`` if not specified.
    """
```